src/components/banner.jsx

```diff
@@ -1,18 +1,20 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { useTranslation } from 'react-i18next';
 import { Grid, GridContainer } from '@trussworks/react-uswds';
 import { LanguageSelector } from '.';
 import nycLogo from '../images/logos/nyc_logo.png';
```

Comment:
Should more general file names be used for images (i.e.`logo.png`)? It will make our app more generalized (not only for CTO and NYC agencies).

Reply:
The image is the nyc logo, so I don't think it makes sense to change the file name. We could change the variable name from `nycLogo` to `logo` though

Reply:
What if this app becomes more readily used by the public (not only City agencies)? Then, they would have to make their file name `nyc_logo` as well for this to work. Otherwise, they would have to go into the code and change it themselves manually.

Reply:
See my comment on header.jsx about creating a file for image exports. Even if we change this file to logo, then we're still asking the user to update their file names to match ours exactly. We can have the exports file with general variable names and where the user just updates the paths to match their file names?

Reply:
I changed to more general file names, as you suggested, and also included the `images/index.js` file for exporting images